modules/ROOT/pages/_partials/installing-mobile-services.adoc

```diff
@@ -29,6 +29,11 @@ $ firewall-cmd --permanent --zone dockerc --add-port 443/tcp
 $ firewall-cmd --reload
 ----
 
+WARNING: Firewall issues may arise on Linux if you try to interact with services on a local cluster from an external device. In order to overcome these issues, you can add the following ports to your firewall.
```

Comment:
I would add something along the lines of "depending on the port your service uses, an example of the ports you may want to add to your firewall are:"